runtime/src/master.rs

```diff
@@ -159,6 +169,8 @@ decl_event! {
         Executed(Vec<Did>, Box<Call>),
```

Comment:
Consider using `Executed(Vec<Did>, Box<Call>, DispatchResult)` and removing `ExecutionFailed` similar to [sudo](https://crates.parity.io/src/pallet_sudo/lib.rs.html#229).

Will make implementation cleaner.